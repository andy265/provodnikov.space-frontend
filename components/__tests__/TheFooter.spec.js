const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const TheFooter = require('~/components/TheFooter')

describe('TheFooter.vue', () => {
  it('has same HTML structure', () => {
    let Component = Object.assign({}, TheFooter, { components: {} })
    let wrapper = shallow(Component)
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
