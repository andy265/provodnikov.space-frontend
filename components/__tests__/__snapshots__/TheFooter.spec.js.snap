// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter.vue has same HTML structure 1`] = `
<footer class="the-footer">
    <p>
        <AppLink to="/">На главную</AppLink>
        |
        <AppLink to="/list">Список постов</AppLink>
        <br>© 2017
        <AppLink to="/">Андрей Проводников</AppLink>
        andrey@provodnikov.space</p>
</footer>
`;
